Marlin/configuration_store.cpp

```diff
@@ -200,16 +206,28 @@ void Config_Postprocess() {
     // Software endstops depend on home_offset
     LOOP_XYZ(i) update_software_endstops((AxisEnum)i);
   #endif
+
+  #if ENABLED(ENABLE_LEVELING_FADE_HEIGHT)
+    set_z_fade_height(
+      //#if ENABLED(AUTO_BED_LEVELING_UBL)
+      //  ubl.state.g29_correction_fade_height
+      //#else
+        planner.z_fade_height
+      //#endif
+    );
```

Comment:
@Roxy-3D This is called both to update the reciprocal value and to correct the current position, if needed.